### PR TITLE
Fixed #26973 -- Fixed views.static.serve() crash with show_indexes enabled.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/26973

### CODE REVIEW

File: django/views/static.py [hunk 1]
```diff
@@ -97,7 +96,6 @@ def serve(request, path, document_root=None, show_indexes=False):
   </body>
 </html>
 """
-template_translatable = ugettext_lazy("Index of %(directory)s")
```
Comment (charettes) [inline django/views/static.py]: It looks like `ugettext_lazy` is now unused.

**Comment by claudep** [comment]
Or would it be possible to feed the libraries argument to `engine()`? Did you plan to write a test for that?

**Comment by timgraham** [comment]
I didn't care to spend much time on this issue as I think this will be deprecated as described in https://groups.google.com/d/msg/django-developers/N0KbgDeLuUE/THUq64NjDwAJ.

If you want to try an alternate patch and write a test, I won't oppose it.

**Comment by claudep** [comment]
Wow nice, many thanks for taking care of this!

**Pull request merged**